### ISSUE TITLE
Add Facet for Object Class Data Types

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-marklogic#141]
**Problem Description:**
A facet to filter for Object data types such as physical object/digital object in Simple Search or Advanced Search do not currently exist.

**Expected Behavior/Solution:**
Create facet/results for the Object scope and all its values in both Simple and Advanced Search.
These results will pull from the "Type" listed in each JSON event record.

Example of record with Physical Object Type:
```
{
"id": "https://lux-front-tst.collections.yale.edu/data/object/64a6f0c2-6d58-44d8-8ffe-43ff5d213b62",
"type": "HumanMadeObject",
"shows": [
{
```
Help text will be added for each value selection. 

**Requirements:**
The following values will be pulled from the Type value in the JSON record:
- value: Physical Object - 'HumanMadeObject',
- value: Digital Object - 'DigitalObject'
- The following help text will be added for each value selection: 
   -  **Object Class:** Search for records categorized as either a "Physical Object" or a "Digital Object"
   - **Physical Object**: Search for Objects that are physical items, such as paintings, sculptures, fossils and other specimens, and archival objects. Individual physical copies of books are included in this class. 
   -  **Digital Object:** Search for Objects that are digital items, such as internet resources (datasets, e-books, and other digital files).  Digitized images of collection items that are physical objects can be found with the “Is Online” facet on Physical Objects
 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- ~[ ] Wireframe/Mockup - Mike~
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - TBD
- ~[ ] Frontend requirements- TBD~
- [ ] ~Are new regression tests required for QA - Amy | Yes~
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- Please check UAT in FE ticket: https://github.com/project-lux/lux-frontend/issues/178

**Dependencies/Blocks:**
Blocking: https://github.com/project-lux/lux-frontend/issues/178

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**

- These links can consist of resources, bugherds, etc.

**Wireframe/Mockup:**
~Place wireframe/mockup for the proposed solution at end of ticket.~

**Comment by roamye** [comment]
Help text has been added in the issue: 
label - Object Class:
 Search for records categorized as either a "Physical Object" or a "Digital Object"

**Comment by roamye** [comment]
Approved by UAT - however this tix will be closed on the 07/08 cycle when the FE can be tested.

**Comment by roamye** [comment]
In prod, the selection of a Object class: digital object or any value does not update for the help text provided above.

after discussion with @clarkepeterf This is because this functionality is not supported at the moment. A new ticket will be created for the new functionality to be supported.

Besides this everything else looks good in PROD. Closing.

**Comment by roamye** [comment]
@clarkepeterf - spoke to sarah and jeff and instead of adding this new functionality to show help text for each value of a class it is better to update the class's help text to include the information of each value. 


Ex: 

Object Class: 
Search for records categorized as either a "Physical Object" or a "Digital Object".

Physical Objects are physical items, such as paintings, sculptures, fossils and other specimens, and archival objects. Individual physical copies of books are included in this class.

Digital Object digital items, such as internet resources (datasets, e-books, and other digital files). Digitized images of collection items that are physical objects can be found with the “Is Online” facet on Physical Objects
